i18n/zh_TW.po

```diff
@@ -841,22 +720,266 @@ msgstr ""
 msgid "Contact Us"
-msgstr ""
+msgstr "聯絡我們"
 
-#: components/AppLayout/AppHeader.js:205
+#: components/AppLayout/AppHeader.js:225
 msgid "My Profile"
-msgstr ""
+msgstr "我的主頁"
 
 #: components/AppLayout/AppSidebar.js:111
 msgid "Forum"
-msgstr ""
+msgstr "討論區"
 
 #: components/AppLayout/AppFooter.js:111
 msgid "Facebook forum"
-msgstr ""
\ No newline at end of file
+msgstr "Facebook 社團"
+
+#: constants/replyType.js:11
+msgid "invalid request"
+msgstr "不在查證範圍"
+
+#: constants/replyType.js:12
+msgid "contains personal perspective"
+msgstr "含有個人意見"
+
+#: constants/replyType.js:13
+msgid "contains true information"
+msgstr "含有正確訊息"
+
+#: constants/replyType.js:14
+msgid "contains misinformation"
+msgstr "含有錯誤訊息"
+
+#: components/ArticleItem.js:152
+msgid "Bust Hoaxes"
+msgstr "查核闢謠"
+
+#: pages/search.js:80
+msgid "Messages"
+msgstr "可疑訊息"
+
+#: components/ArticlePageLayout.js:234
+msgid "Replied by me"
+msgstr "我查核過"
+
+#: components/ArticlePageLayout.js:235
+msgid "No useful reply yet"
+msgstr "還未有有效查核"
+
+#: components/ArticlePageLayout.js:236
+msgid "Asked many times"
+msgstr "熱門回報"
+
+#: components/ArticlePageLayout.js:237
+msgid "Replied many times"
+msgstr "熱門討論"
+
+#: components/ArticlePageLayout.js:251
+#: components/ArticlePageLayout.js:488
+msgid "Filter"
+msgstr "篩選"
+
+#: components/ArticlePageLayout.js:282
+msgid "Topic"
+msgstr "主題"
+
+#: components/ArticlePageLayout.js:286
+msgid "All Topics"
+msgstr "全部主題"
+
+#: components/ArticlePageLayout.js:474
+msgid "Load More"
+msgstr "查看更多"
+
+#: pages/hoax-for-you.js:25
+#: pages/hoax-for-you.js:38
+msgid "Hoax for you"
+msgstr "等你來打"
+
+#: pages/replies.js:21
+msgid "Latest replies"
+msgstr "最新查核"
+
+#: components/ExpandableText.js:31
+msgid "Show Less"
+msgstr "收合全文"
+
+#: components/ExpandableText.js:31
+msgid "Show More"
+msgstr "閱讀全文"
+
+#: components/FeedDisplay.js:53
+msgid "Subscribe"
+msgstr "訂閱本頁"
+
+#: components/Filters.js:167
+msgid "Collapse"
+msgstr "收合"
+
+#: components/Filters.js:172
+msgid "Expand"
+msgstr "展開"
+
+#: components/ReplyFeedback.js:318
+msgid "See Reasons"
+msgstr "看理由"
+
+#: components/ReplyFeedback.js:349
+#, javascript-format
+msgid "Helpful ${ positiveFeedbackCount }"
+msgstr "有幫助 ${ positiveFeedbackCount }"
+
+#: components/ReplyFeedback.js:353
+msgid "Not Helpful ${ negativeFeedbackCount }"
+msgstr "沒有幫助 ${ negativeFeedbackCount }"
+
+#: components/ReplyFeedback.js:416
+msgid "Send"
+msgstr "送出"
+
+#: components/ReplyItem.js:89
+msgid "Anonymous"
+msgstr "有人"
+
+#: components/ReplyItem.js:102
+#, javascript-format
+msgid "${ userName } consider this ${ TYPE_NAME[replyType] }"
+msgstr "${ userName } 認為 ${ TYPE_NAME[replyType] }"
+
+#: components/TimeRange.js:63
+msgid "In 1 Day"
+msgstr "一天內"
+
+#: components/TimeRange.js:64
+msgid "In 1 Week"
+msgstr "一週內"
+
+#: components/TimeRange.js:65
+msgid "In 1 Month"
+msgstr "一個月內"
+
+#: components/TimeRange.js:163
+msgid "to"
+msgstr "到"
+
+#: components/ArticleCategories/AddCategoryDialog.js:86
```

Comment:
These are temporary components that is going to be replaced, so I don't translate these.